openpype/hosts/houdini/plugins/inventory/set_camera_resolution.py

```diff
@@ -0,0 +1,25 @@
+from openpype.pipeline import InventoryAction
+from openpype.hosts.houdini.api.lib import (
+    get_camera_from_container,
+    set_camera_resolution
+)
+
+
+class SetCameraResolution(InventoryAction):
+
+    label = "Set Camera Resolution"
+    icon = "desktop"
+    color = "orange"
+
+    @staticmethod
+    def is_compatible(container):
+        print(container)
+        return (
+            container.get("loader") == "CameraLoader"
+        )
+
+    def process(self, containers):
+        for container in containers:
+            node = container["node"]
+            camera = get_camera_from_container(node)
+            set_camera_resolution(camera)
```

Comment:
Query the document from database only once.
```suggestion
        asset_doc = get_current_project_asset()
        for container in containers:
            node = container["node"]
            camera = get_camera_from_container(node)
            set_camera_resolution(camera, asset_doc)
```